apps/serdes/halide_ir.fbs

```diff
@@ -0,0 +1,416 @@
+namespace Halide.Serdes;
+
+// This corresponds to the version.
+file_identifier "HLD0";
```

Comment:
Is the identifier required to be exactly four characters?

Reply:
Yes, from flatbuffers' docs:
```
Now, you can always prefix a FlatBuffer with your own file header, but FlatBuffers has a built-in way to add an identifier to a FlatBuffer that takes up minimal space, and keeps the buffer compatible with buffers that don't have such an identifier.

You can specify in a schema, similar to root_type, that you intend for this type of FlatBuffer to be used as a file format:

file_identifier "MYFI";
Identifiers must always be exactly 4 characters long. These 4 characters will end up as bytes at offsets 4-7 (inclusive) in the buffer.
```
https://flatbuffers.dev/md__schemas.html